dev/docker-compose.dev.messagingservice.yaml

```diff
@@ -3,7 +3,7 @@
 version: "3.7"
 services:
   messagingservice:
-    command: sh -c "cd /opt/app && flask run --host 0.0.0.0 --port ${PORT:-8000}"
+    command: sh -c "cd /opt/app && flask upgrade && flask run --host 0.0.0.0 --port ${PORT:-8000}"
```

Comment:
I found it helpful to run fewer things on startup when developing cPRO, but no strong feelings